pydgraph/txn.py

```diff
@@ -56,9 +56,9 @@ def __init__(self, client, read_only=False, best_effort=False):
         self._read_only = read_only
         self._best_effort = best_effort
 
-    def query(self, query, variables=None, timeout=None, metadata=None, credentials=None):
+    def query(self, query, variables=None, timeout=None, metadata=None, credentials=None, resp_format="json"):
```

Comment:
@danielmai I am not sure if this is the correct way to add the optional param. This should be in a format that proto understands (enum in this case). What do you suggest?